components/camel-workday/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>components</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>camel-workday</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Camel :: Workday</name>
+  <description>Camel Workday support</description>
+
+  <properties>
+    <camel.osgi.export.pkg>org.apache.camel.component.workday.*</camel.osgi.export.pkg>
+    <camel.osgi.export.service>org.apache.camel.spi.ComponentResolver;component=workday</camel.osgi.export.service>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>commons-httpclient</groupId>
```

Comment:
Please use a newer HTTP client like httpcomponents, the last release of commons-httpclient is from 2007

Reply:
Changed to HttpComponents-client,